app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -116,31 +116,35 @@ function trackCustomImpressions() {
     }
```

Comment:
I've noticed that this file has a few eslint related errors: lacking of `use strict`, some unnamed functions and so on. Do you think it's worth it cleaning it up in this PR?

Reply:
I often do this is a smaller follow up PR, the "Reference in new issue" option from the 3 dot menu on a comment makes this very easy to track.